scss/elements/_lists.scss

```diff
@@ -19,7 +19,9 @@
 ul,
 ol {
   list-style-type: none;
-  margin: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+  margin-left: .25rem;
```

Comment:
Do you just want this to apply to bulletted lists? we often use `ul` for things that we don't want to look like lists, so if this is for bullet's it should go under `list--bulleted` in `list-styles.scss`.